File: Codeforces rounds/Codeforces rounds/Codeforces Round 173 (Div. 2)/A.py
# https://codeforces.com/contest/282/problem/A


def main():

    n: int = int(input())

    x: int = 0

    for _ in range(n):
        s: str = input()
        if s[1] == '+':
            x += 1
        else:
            x -= 1

    print(x)


if __name__ == '__main__':
    main()

File: Codeforces rounds/Codeforces rounds/Codeforces Round 173 (Div. 2)/A.cpp
// https://codeforces.com/contest/282/problem/A

#include <bits/stdc++.h>

using namespace std;

typedef unsigned int uint;

int main(void) {
  ios_base::sync_with_stdio(false);
  cin.tie(NULL);

  uint n;
  cin >> n;

  int x = 0;

  for (uint i = 0; i < n; i++) {
    char a, b, c;
    cin >> a >> b >> c;

    if (b == '+') {
      x++;
    } else {
      x--;
    }
  }

  cout << x << '\n';
}
